Training24/ClassicLibrary.c

```diff
@@ -3,9 +3,7 @@
 // Copyright (c) Metamation India.
 // Kalaimagal V P
 // ------------------------------------------------------------------------------------------------
-// ClassicLibrary.c
-// Program on A5 branch.
-// ClassicLibrary.c - The program sorts numbers and searches for a specific number in the sorted list.
+// // ClassicLibrary.c - The program sorts numbers and searches for a specific number in the sorted list.

```

Comment:
```suggestion
// ClassicLibrary.c - The program sorts numbers and searches for a specific number in the sorted list.

```
Please use git di and examine your changes. You will be able to avoid mistakes like this.